src/app/shared/shared.module.ts

```diff
@@ -4,9 +4,8 @@ import { CommonModule } from '@angular/common';
 import { NgxPaginationModule } from 'ngx-pagination';
 import { SearchResultComponent } from './search/search-result.component';
 import { SearchFiltersComponent } from './search/filters/search-filters.component';
-import { PaletteModule, CaseFieldService, ConditionalShowModule, DocumentDialogComponent,
-  MarkdownModule, LabelFieldComponent, LabelSubstitutorModule, SharedUtilsModule,
-  DeleteOrCancelDialogComponent, SaveOrDiscardDialogComponent} from '@hmcts/ccd-case-ui-toolkit';
+import { PaletteModule, CaseFieldService, ConditionalShowModule, MarkdownModule, LabelFieldComponent,
+  LabelSubstitutorModule, SharedUtilsModule, DialogsModule } from '@hmcts/ccd-case-ui-toolkit';
```

Comment:
`DialogsModule` is not imported in the `NgModule` imports. Shouldn't that be the case? otherwise the import above is never used.